streamer/src/nonblocking/quic.rs

```diff
@@ -115,7 +115,9 @@ pub fn spawn_server(
     coalesce: Duration,
 ) -> Result<(Endpoint, Arc<StreamStats>, JoinHandle<()>), QuicServerError> {
     info!("Start {name} quic server on {sock:?}");
-    let (config, _cert) = configure_server(keypair, gossip_host)?;
+    let concurrent_connections = max_staked_connections + max_unstaked_connections;
+    let max_concurrent_connections = concurrent_connections + concurrent_connections / 4;
```

Comment:
i thought we needed to consider the endpoint count here as well, to avoid implicit multiplication

Reply:
The multiquic code is not in yet -- when it is -- we have to deal with it. We only have one Endpoint for now.